exe/ruby-lsp-check

```diff
@@ -47,9 +47,11 @@ index = RubyIndexer::Index.new
 indexables = RubyIndexer.configuration.indexables
 
 indexables.each_with_index do |indexable, i|
-  result = Prism.parse(File.read(indexable.full_path))
-  collector = RubyIndexer::Collector.new(index, result, indexable.full_path)
-  collector.collect(result.value)
+  content = File.read(indexable.full_path)
```

Comment:
@vinistock Was there a reason to not call `index.index_single` here `ruby-lsp-doctor` instead?